examples/aStar path finding algorithm visualizer/README.md

```diff
@@ -0,0 +1,36 @@
+# A Star path finding algorithm visualizer
+This is a simple visualizer template for path finding algorithm wriiten in v lang
+
+# Quick start 
+
+make sure you have c lang installed or follow the instruction here 
+- v language docs: [v language docs](https://github.com/vlang/v/blob/master/doc/docs.md) 
```

Comment:
There is no need for these lines; all the examples assume that V is installed, and there is a working C compiler (and V distributes a prebuilt tcc for linux/windows/macos).